include/deal.II/lac/utilities.h

```diff
@@ -0,0 +1,160 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2017 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__lac_utilities_h
+#define dealii__lac_utilities_h
+
+#include <deal.II/base/config.h>
+#include <deal.II/base/exceptions.h>
+#include <deal.II/lac/lapack_templates.h>
+#include <deal.II/lac/lapack_support.h>
+#include <deal.II/lac/vector_memory.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+namespace Utilities
+{
+  /**
+   * A collection of linear-algerbra utilities
+   */
+  namespace Lac
```

Comment:
I think that's the first time we use this namespace name. Is this how we want to write this? We already have "namespace LACExceptions", and in general I'm not very fond of the abbreviation "LAC" anyway -- we imported that term from the DEAL library that predated deal.II. Maybe name it "namespace LinearAlgebra"?